scripts/install/ceres_install.bash

```diff
@@ -2,6 +2,7 @@
 set -e  # exit on first error
 UBUNTU_VERSION=`lsb_release --release | cut -f2`
 SRC_PREFIX_PATH="/usr/local/src/"
```

Comment:
Indeed they can get big but they also save time during compilation, so it is 50/50 decision to remove it or not, I opted not to by default because space is rarely an issue for modern systems, of course there are exceptions.